### PR TITLE
Add only code coverage for unit testing

### PR DESCRIPTION
**Pull request by rohitsakala**
[#164955500](https://www.pivotaltracker.com/n/projects/2192232/stories/164955500)

Code coverage report can be accessed at https://coveralls.io/github/cloudfoundry-incubator/cf-operator

### CODE REVIEW

**Comment by viovanov** [comment]
@rohitsakala I don't think this is what @manno meant
it looks like you're ignoring some suites - I believe @manno meant to ignore specific code directories

**Comment by manno** [comment]
Does removing `-cover` remove the coverage reporting altogether?

File: bin/test-integration [hunk 1]
```diff
@@ -25,7 +25,12 @@ bin/apply-crds
...
 kubectl get customresourcedefinitions
 
-pkgs=$(go list ./... 2> /dev/null)
-deps=`echo ${pkgs} | tr ' ' ","`
-ginkgo --slowSpecThreshold=50 -cover -coverpkg "$deps" integration/
-
+#deps="code.cloudfoundry.org/cf-operator/cmd,code.cloudfoundry.org/cf-operator/pkg"
+ginkgo --slowSpecThreshold=50 --cover --coverpkg code.cloudfoundry.org/cf-operator/pkg/... integration/
+
+# Send code coverage results to coveralls.io
```
Comment (manno) [inline bin/test-integration]: Can we move this into a separate file? I run this script locally a lot.

Comment (rohitsakala) [reply bin/test-integration]: Hi @manno , sure, I was also thinking the same. I am not yet done. I am testing a few things. will be done by tonight.

**Pull request merged**